cypress/integration/userCanOrderProduct.feature.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
+describe("", () => {
+  before(() => {
+    cy.intercept("GET", "**/api/products", { fixture: "products.json" });
+    cy.intercept("POST", "**/api/orders", {
+      fixture: "orderCreateResponse.json",
+    }).as("Orders.create");
+    cy.visit("/");
+    cy.get("[data-cy=product-list]")
+      .children()
+      .first()
+      .within(() => {
+        cy.get("button").click();
+      });
+  });
+
+  it("is expected to make a POST request", () => {
+    cy.wait("@Orders.create").its("request.method").should("eq", "POST");
+  });
+});
```

Comment:
in the fixture file you have a message "Pizza was added to your order" not testing for this?

Reply:
Either test for it or don't have it in the fixture?

Reply:
We need to write a test for that.

Reply:
I removed the fixture. When we have written the test we re-add it.